AllReadyApp/Web-App/AllReady/Security/ClaimsExtensions.cs

```diff
@@ -1,13 +1,20 @@
 ﻿using System;
 using System.Linq;
 using System.Security.Claims;
+using System.Threading.Tasks;
 using AllReady.Models;
 using Microsoft.AspNetCore.Identity;
 
 namespace AllReady.Security
 {
     public static class ClaimsExtensions
     {
+        public static async Task<string> GetUserId(this ClaimsPrincipal claimsPrincipal, UserManager<ApplicationUser> userManager)

```

Comment:
@stevejgordon, you're right, we don't need it. I added it as a way for contributors to quickly be able to get the current user's user id w/out knowledge of user manager.... this might, or might not be a good thing.

For now, I'll remove the extension method and just reach out to UserManager directly in the code. 

Thanks for the feedback